content/containers.md

```diff
@@ -21,6 +21,8 @@ Since every app has unique architecture and performance characteristics, adjusti
 - Double: 2 GB/2 ECU
 - Quad: 4 GB/4 ECU
 
+[ECU units](https://aws.amazon.com/ec2/faqs/) are defined by Amazon and are designed to provide a relative measure of processing power.
```

Comment:
Can link directly to https://aws.amazon.com/ec2/faqs/#hardware-information and maybe expand the abbreviation too.